python/ambassador-diag/src/ambassador_diag/diagd.py

```diff
@@ -64,9 +79,7 @@
     parse_bool,
     parse_json,
 )
-
-if TYPE_CHECKING:
-    from ambassador.ir.irtlscontext import IRTLSContext  # pragma: no cover
+from ambassador_diag.templates import TEMPLATE_PATH
```

Comment:
When I run `gmake lint/mypy` in this branch _before_ you make this edit, `mypy` doesn't flag 

```
if TYPE_CHECKING:
    from ambassador.ir.irtlscontext import IRTLSContext  # pragma: no cover
```

as needing attention, but it does say 

```
pyproject.toml: [mypy]: Unrecognized option: show_none_errors = False
```

So... what are you running? 🙂

Reply:
Ew. On `dev/v4` before this change, `mypy` _also_ doesn't flag this line. 😛

Reply:
I fixed that `show_none_errors` warning (copy/paste error). Uhhh... not sure about the lack of flagging though?